wagtail/wagtailadmin/templates/wagtailadmin/notifications/base.html

```diff
@@ -0,0 +1,133 @@
+{% load i18n static %}
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title></title>
+<style>
+{% block styles_content %}
+body {
+	min-width:100%;
+	-webkit-text-size-adjust: none;
+	padding: 0;
+	margin: 0;
+}
+.ReadMsgBody {
+	width: 100%;
+}
+.ExternalClass {
+	width: 100%;
+}
+table {
+	border-collapse:collapse;
+	mso-table-lspace:0pt;
+	mso-table-rspace:0pt;
+}
+table td {
+	border-collapse:collapse;
+}
+td.date a:link {
+	color:#6b6b6b;
+	text-decoration:none;
+}
+@media only screen and (max-width: 568px) {
+body[yahoo] .mobile {
+	width:320px !important;
+}
+body[yahoo] .spacer {
+	width:10px !important;
+}
+body[yahoo] .spacer_height {
+	height:10px !important;
+}
+body[yahoo] .content {
+	width:300px !important;
+}
+body[yahoo] .col-3{
+  float: none !important;
+	width: 100% !important;
+}
+body[yahoo] .force-col {
+	display: block;
+	padding-right: 0 !important;
+	padding-left: 0 !important;
+	padding-bottom: 12px !important;
+}
+body[yahoo] .force-col-last {
+	display: block;
+	padding-right: 0 !important;
+	padding-bottom: 0 !important;
+}
+body[yahoo] .center {
+	text-align:center;
+}
+body[yahoo] .show {
+	display: block !important;
+}
+body[yahoo] .hide {
+	display: none !important;
+}
+body[yahoo] .img {
+	width: 130px !important;
+	float:left;
+}
+body[yahoo] .img img {
+	width:100% !important;
+}
+body[yahoo] .text {
+	width: 150px !important;
+	float:right;
+}
+{% endblock %}
+</style>
+</head>
+<body style="background-color:#E6E6E6;padding:0px;margin:0px;" yahoo="fix" topmargin="0" bottommargin="0" leftmargin="0" rightmargin="0">
+
+<table width="600" border="0" align="center" class="mobile" cellspacing="0" cellpadding="0">
+  <tr>
+    <td>
+    	{% block branding_logo %}<img width="100%" border="0" style="display: block;" alt="" src="{{ settings.BASE_URL }}{% static 'wagtailadmin/images/email-header.jpg' %}" />{% endblock %}
```

Comment:
I had some problems with images in emails, but I don't remember details... Are you sure that email will not be blocked by email providers because of remote image?